src/Compilers/Shared/DesktopBuildClient.cs

```diff
@@ -157,12 +157,12 @@ protected override int RunLocalCompilation(List<string> arguments, string client
                 }
             }
 
-            return null;
+            return Task.FromResult<BuildResponse>(null);
```

Comment:
Why does this method have to return `Task<BuildResponse>` instead of just `BuildResponse`?  There are no `await`'s here.  Or is it to make the signature equivalent to another function that does use `await`?